distributed_shampoo/utils/shampoo_preconditioner_list.py

```diff
@@ -1147,6 +1151,9 @@ def _create_kronecker_factors_state_for_block(
             )
             for dim in preconditioned_dims
         )
+        # Initialize factor_matrices_eigenvalues all ones.
+        for t in factor_matrices_eigenvalues:
+            block_info.get_tensor(t).fill_(1.0)
```

Comment:
This could also be implemented with a new `block_info.allocate_ones_tensor`method.